zeeguu/core/word_scheduling/basicSR/basicSR.py

```diff
@@ -232,6 +265,8 @@ def schedule_some_more_bookmarks(cls, session, user, required_count):
             id = b["bookmark_id"]
             b = Bookmark.find(id)
             print(f"scheduling another bookmark_id for now: {id} ")
+            b.learning_cycle = LearningCycle.RECEPTIVE.value
```

Comment:
Since I got errors without using `.value`, we might want to use this syntax for all enums, where we are using Enums. I remember we discussed it might be due to the new version of python? Maybe @mircealungu can check if he gets any errors.